.github/ISSUE_TEMPLATE/german_localization.md

```diff
@@ -0,0 +1,23 @@
+---
+name: IO übersetzen
+about: Aiutaci a tradurre l'app in tedesco
+title: ''
+labels: "internationalization", "German :de:"
```

Comment:
I'd user `localization` since it relates to the content rather than the infrastructure.

Reply:
I was referring to the label mentioned in the general redme file, but if "internationalization" is intended to be used for 'nerdy' stuff, then I'm okay having only "German 🇩🇪" as label 😂

Reply:
jokes apart: I'm thinking about the future where we may migrate to a different platform thus separating content from logic. 
It would be useful to be able to group pure content-related issues as opposed to technical ones using one filter.